nextcord/abc.py

```diff
@@ -1411,6 +1432,12 @@ async def send(
         channel = await self._get_channel()
         state = self._state
         content = str(content) if content is not None else None
+        if flags is None:
+            flags = MessageFlags()
+        if suppress_embeds is True:
+            flags.suppress_embeds = True
```

Comment:
1. `if x is True` violates PEP 8 recommendations.
2. Accompanied with my previous review comment, why not just set `flags.suppress_embeds` directly to `suppress_embeds`?

Reply:
This should instead simply check for `None` then pass it explicitly, as this relies on the API default - consistency.
```suggestion
        if suppress_embeds is not None:
            flags.suppress_embeds = suppress_embeds
```